CMSgov/easi-app
# [NOREF] GQL Schema for System Intake linking

# NOREF

## Changes and Description

- Introduces several new query and mutation definitions that will define how we'll implement the System Intake linking logic for System-Centric EASi

This does not add any actual resolver logic, but just helps to set the direction that we want to go with these resolvers!

## How to test this change

This shouldn't change any existing logic. Make sure the app compiles, and everything is named / designed as you'd expect!

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
